src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -94,12 +94,12 @@ public Contig apply(String contigString) {
           }
         });
   }
-  
+
   // The following methods have package scope and are helpers for ShardUtils. For sharded Contigs,
   // the ShardUtils methods should be used to ensure that shards are shuffled all together before
   // being returned to clients.
   List<Contig> getShards(long numberOfBasesPerShard) {
-    double shardCount = Math.ceil(end - start) / (double) numberOfBasesPerShard;
+    double shardCount = Math.ceil(end - start) / numberOfBasesPerShard;
```

Comment:
*Nota bene*: Performing a ceiling on the difference between two `long` numbers will not change anything.  I think you might want to have the following - with an additional encompassing set of parentheses - to have the intended effect:

```Java
double shardCount = Math.ceil( (end - start) / numberOfBasesPerShard );
```